### ISSUE TITLE
Style guide build options

### ISSUE THREAD

**Issue by u-v** [dimanech/atlas-guide#68]
Make it possible to pass a style guide configuration object as an optional parameter for build() and buildAll() functions.

.atlasrc.json - will be used as a fallback

**Comment by dimanech** [comment]
Hi @u-v! This feature is implemented and will be available for testing in `2.0.0-0`. So right now different configs could be passed with `withConfig`:

```js
const atlasGuide = require('atlas-guide').withConfig('./project/root/path/to/config.json');
// or atlasGuide = require('atlas-guide').withConfig({ rawConfigObject });
atlasGuide.build().then(...); 
```
also search of configs in project root and package.json is disabled, so it needs to pass config explicitly.

**Comment by u-v** [comment]
cool!!! thanks!
